.github/workflows/publish-npm-packages.yml

```diff
@@ -41,163 +49,12 @@ jobs:
           token: ${{ secrets.NPM_PUBLISH_TOKEN }}
           package: inlang/source-code/cli/package.json
 
-      - name: "@inlang/json-types"
-        uses: JS-DevTools/npm-publish@v2
```

Comment:
Does an action like this exist for pnpm? 

- action checks if version is already published

Reply:
Perfect!